cache/in-memory/src/permission.rs

```diff
@@ -465,7 +472,7 @@ impl<'a> InMemoryCachePermissions<'a> {
 
     fn parent_overwrites(
         &self,
-        channel_id: &ChannelId,
+        channel_id: &Id<ChannelMarker>,
```

Comment:
does this still have to be a borrow? it is a copy type

Reply:
Maybe not, but I think that's a topic for another PR.